__tests__/__snapshots__/no-config.test.js.snap

```diff
@@ -32,14 +32,14 @@ Object {
               "extension": ".js",
               "name": "polyfill.chunk.js",
               "path": "<TEMP_DIR>/_underreact-site/underreact-assets/js/polyfill.chunk.js",
-              "size": 97256,
+              "size": 97446,
```

Comment:
Do you know why the file size changed?

Reply:
I can confirm that this is nothing but harmless jumbling, mostly whitespaces, due to `node_modules` compilation.